screwdriver.yaml

```diff
@@ -0,0 +1,34 @@
+cache:
+    event:
+        - $SD_SOURCE_DIR/node_modules
+    pipeline:
+        - ~/.gradle
+    job:
+        test-job: [/tmp/testblah]
+jobs:
+    test-job:
+        image: node:6
+        steps:
+            - ls-tmp: ls /tmp
+            - echo: echo hi > /tmp/testblah
+        requires: [~commit, ~pr]
+    npm1:
+        image: node:6
+        steps:
+            - install: npm install
+        requires: [~commit, ~pr]
+    npm2:
+        image: node:6
+        steps:
+            - ls: ls
+            - install: npm install
+        requires: [npm1]
+    gradle:
+        image: java:7
+        steps:
+            - ls: ls ~/
```

Comment:
this should possible contain `ls -a ~/` so that hidden folders can be listed (gradle one is)